dev/devicelab/lib/tasks/plugin_tests.dart

```diff
@@ -77,6 +77,11 @@ class PluginTest {
       final _FlutterProject app = await _FlutterProject.create(tempDir, options, buildTarget,
           name: 'plugintestapp', template: 'app', environment: appCreateEnvironment);
       try {
+        if (cocoapodsTransitiveFlutterDependency) {
+          section('Disable Swift Package Manager');
+          await app.disableSwiftPackageManager();
```

Comment:
We update the build settings of pods that depend on the Flutter framework (including transitive dependencies). This test only makes sense for CocoaPods, so we need to ensure we use CocoaPods.

For more context, see: https://github.com/flutter/flutter/pull/125610



 If a CocoaPo
We update CocoaPod 
This